types/src/on_chain_config/consensus_config.rs

```diff
@@ -217,6 +239,14 @@ impl OnChainConsensusConfig {
         }
     }
 
+    pub fn order_vote_enabled(&self) -> bool {
+        match &self {
+            OnChainConsensusConfig::V1(_config) => false,
+            OnChainConsensusConfig::V2(_) => false,
+            OnChainConsensusConfig::V3 { .. } => false,
```

Comment:
this is supposed to return true when it's enabled?

Reply:
Don't we disable the flag by default and enable the flag via a proposal?

Reply:
yes, but this code makes proposal irrelevant? whatever value we set in proposal returns false here